package.json

```diff
@@ -87,5 +87,9 @@
   },
   "msw": {
     "workerDirectory": "public"
+  },
+  "volta": {
+    "node": "18.14.2",
```

Comment:
**local-ssl-proxy** work in this node@18 version ? please let me know 
[See this](https://github.com/cameronhunter/local-ssl-proxy/issues/98)

Reply:
its working now